### PR TITLE
deepks : unit test & clean up the code

### PR DESCRIPTION
**Pull request by wenfei-li**

### CODE REVIEW

File: source/module_deepks/LCAO_deepks_psialpha.cpp [hunk 1]
```diff
@@ -154,10 +154,10 @@ void LCAO_Deepks::check_psialpha(const bool& calc_deri,
     std::ofstream ofs_y("dpsialpha_y.dat");
     std::ofstream ofs_z("dpsialpha_z.dat");
 
-    ofs<<std::setprecision(12);
-    ofs_x<<std::setprecision(12);
-    ofs_x<<std::setprecision(12);
-    ofs_x<<std::setprecision(12);
+    ofs<<std::setprecision(10);
+    ofs_x<<std::setprecision(10);
+    ofs_x<<std::setprecision(10);
```
Comment (caic99) [inline source/module_deepks/LCAO_deepks_psialpha.cpp]: Maybe `ofs_y` here? And 'z' below?

File: source/module_deepks/test/LCAO_deepks_test.cpp [hunk 2]
```diff
@@ -248,4 +324,35 @@ void test_deepks::check_f_delta()
...
+	bool same=true;
+
+	std::string word1;
+	std::string word2;
+	while(file1 >> word1)
+	{
+		file2>>word2;
+		if(word1[0]-'0'>=0||word1[0]-'0'<10||word1[0]=='-')
+		{
+			if(word1!=word2) same=false;
```
Comment (caic99) [inline source/module_deepks/test/LCAO_deepks_test.cpp]: So, there might be cases when input value is not a number, but string?

Comment (wenfei-li) [reply source/module_deepks/test/LCAO_deepks_test.cpp]: yes, but it won't start with number or minus sign

File: source/module_deepks/LCAO_deepks_psialpha.cpp (hunk 1 above)
Comment (wenfei-li) [reply source/module_deepks/LCAO_deepks_psialpha.cpp]: definitely, thanks!

**Pull request merged**